scipy/signal/setup.py

```diff
@@ -30,7 +30,8 @@ def configuration(parent_package='', top_path=None):
 
```

Comment:
There's a second `PythranExtension` in this file, for `_max_len_seq_inner`. Could you add `config=` there too?

Other than that, this looks good to me.